bundles/scripting/sightly/js-use-provider/pom.xml

```diff
@@ -70,6 +70,7 @@
                     <instructions>
                         <Sling-Bundle-Resources>/libs/sling/sightly/js;path:=/SLING-INF/libs/sling/sightly/js</Sling-Bundle-Resources>
                         <Require-Capability>io.sightly; filter:="(&amp;(version&gt;=1.0)(!(version&gt;=2.0)))"</Require-Capability>
+                        <Import-Package>org.mozilla.javascript;version="[1.1,2)",*</Import-Package>
```

Comment:
Because the import for `org.mozilla.javascript` is not resolved via a bundle (it's resolved directly via `org.mozilla.rhino-1.7R4`) the bnd plugin will not generate a version range for the import. If you also have `org.mozilla.javascript' version 2+ on the same Sling instance, the package import will then resolve to the higher/incompatible package.